### DISCUSSION TITLE
how to let afl save timeout testcase

### DISCUSSION THREAD

**Discussion by hac425xxx** [AFLplusplus/AFLplusplus#2011] [General]
I'm fuzzing some program,it has many timeout case, I want to debug these testcase, how to let afl save timeout testcase for me?

![image](https://github.com/AFLplusplus/AFLplusplus/assets/56855299/8b792028-2072-497f-ae14-1089b809e3a8)

**Comment by vanhauser-thc** [comment]
it automatically saved them when the timeouts are sever (x2 of configured timeout), they are then called "hangs".
you can reduce the timeout (with -t) and then easier create hangs. or increase timeout and have those timeouts also complete (as they do not take that much longer then the set timeout)
